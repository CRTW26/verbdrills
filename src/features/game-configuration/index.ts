export { default as GameConfigurationForm } from './components/ConfigurationForm'
