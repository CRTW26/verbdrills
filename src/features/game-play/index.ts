export { default as GamePlay } from './components/GamePlay'
