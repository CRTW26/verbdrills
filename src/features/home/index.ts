export { default as TitleBar } from './components/TitleBar'
export { default as Description } from './components/Description'
