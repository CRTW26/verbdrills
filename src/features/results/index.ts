export { default as Results } from './components/Results'
export { default as IncorrectAnswers } from './components/IncorrectAnswers'
