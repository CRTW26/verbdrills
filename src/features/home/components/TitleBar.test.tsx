import '@testing-library/jest-dom'
import { render } from '@testing-library/react'
import { TitleBar } from 'features/home'

describe('<TitleBar>', () => {
  test('should render', () => {
    const { asFragment } = render(<TitleBar text="verbdrills" />)

    expect(asFragment()).toMatchSnapshot()
  })

  test('should have correct heading', () => {
    const { getByTestId } = render(<TitleBar text="verbdrills" />)

    expect(getByTestId('title-bar')).toHaveTextContent('verbdrills')
  })
})
