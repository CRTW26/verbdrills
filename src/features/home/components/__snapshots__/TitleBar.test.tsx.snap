// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TitleBar> should render 1`] = `
<DocumentFragment>
  <div
    class="jsx-964221634 title-bar"
    data-testid="title-bar"
  >
    <h1
      class="jsx-964221634"
    >
      verbdrills
    </h1>
    <h2
      class="jsx-964221634"
    >
      Master verb conjugation
    </h2>
  </div>
</DocumentFragment>
`;
